README.md

```diff
@@ -55,3 +55,19 @@ To iterate on the database:
 - You can explore the database with `yarn drizzle-kit studio`
 
 After the initial iterations, we'll start using migrations (`yarn drizzle-kit generate` & `yarn drizzle-kit migrate`)
+
+## Database Migration:
+
+After we update the schema in `packages/nextjs/services/database/config/schema.ts`, we can generate a migration with:
+
+```
+yarn drizzle-kit generate
+```
+
+Then we can apply the migration with:
+
+```
+yarn drizzle-kit migrate
+```
+
+We also need to make sure we commit the migration to the repo.
```

Comment:
Happy to update this in other way, if we want 🙌

Reply:
Should we tweak the order here (first migrations, second "push")

If someone is setting the environment, I guess we want them to use `migrate` instead of `push`, right?

(and later on, we can hint that you can use `push` to tinker, and then run `generate` to create the migration).

Let me know if you need help.

Thanks!

Reply:
Ohhh yess!!! Just pushed some tweaks but please let me know if I should tweak more 🙌